tests/test_logger.py

```diff
@@ -0,0 +1,33 @@
+#  Copyright 2021 EPAM Systems
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import utils
+from six.moves import mock
+
+from tests.conftest import REPORT_PORTAL_SERVICE
+
+
+@mock.patch(REPORT_PORTAL_SERVICE)
+def test_launch_log(mock_client_init):
+    utils.run_robot_tests('examples/launch_log.robot')
```

Comment:
Maybe we should distribute modules to the appropriate folder. This particular test case looks like a complex integration test rather than a unit one. Let's put integration tests into tests/integration folder, and unit tests into tests/unit?